src/main/rx/spektrum.c

```diff
@@ -210,52 +207,80 @@ bool spekShouldBind(uint8_t spektrum_sat_bind)
  */
 void spektrumBind(rxConfig_t *rxConfig)
 {
-    int i;
     if (!spekShouldBind(rxConfig->spektrum_sat_bind)) {
         return;
     }
 
-    LED1_ON;
+    ioTag_t bindPin;
+
+    if (rxConfig->spektrum_bind_pin_override_ioTag) {
+        bindPin = rxConfig->spektrum_bind_pin_override_ioTag;
+    } else {
+        const serialPortConfig_t *portConfig = findSerialPortConfig(FUNCTION_RX_SERIAL);
+        if (!portConfig) {
+            return;
+        }
 
-    BindPin = IOGetByTag(IO_TAG(SPEKTRUM_BIND_PIN));
-    IOInit(BindPin, OWNER_RX_BIND, 0);
-    IOConfigGPIO(BindPin, IOCFG_OUT_PP);
+        int index = SERIAL_PORT_IDENTIFIER_TO_INDEX(portConfig->identifier);
+        ioTag_t txPin = serialPinConfig()->ioTagTx[index];
+        ioTag_t rxPin = serialPinConfig()->ioTagRx[index];
+
+        switch (rxConfig->serialrx_provider) {
+        case SERIALRX_SRXL:
+#ifdef TELEMETRY
```

Comment:
Is it really needed to have a compile condition on TELEMETRY ? And also the next line a runtime check for the TELEMETRY feature being turned on? Binding has no dependency to TELEMETRY as such. Of course you have to turn on TELEMETRY for SRXL to work fully and return telemetry data, but that has nothing to do with binding. 
Unnecessary conditions and constraints might/will lead to support questions and issues.